docs/FybrikNetworkTopologiesAndRequirements.md

```diff
@@ -0,0 +1,78 @@
+# Fybrik data plane network connection types and isolation requirements
+
+## Introduction
+This document explains Fybrik data plane network topologies and isolation requirements.
+We have started from the data plane because it is dynamically created and might include third-part developed components.
+Furthermore, user data is accessed, modified and moved by the data plane only.
+Therefore, networking isolation and protection of data plane is very important and challenged task, and we start from it.
+
+### Types of Fybrik Network Connections
+Fybrik data plane can be spread across several Kubernetes clusters and theoretically include any number of modules. 
+The following picture demonstrates different Fybrik data plane connection types.
+![](./images/ConnectionTypes.png)
+
+User applications (workloads) achieve the Fybrik data plane via its entry point. Depends on the location of the
```

Comment:
Depending

Reply:
Better to rephrase: "There are 3 different connection types, based on the location of the application:

Reply:
OK